### DISCUSSION TITLE
Possible to disable SSL certificate verification?

### DISCUSSION THREAD

**Discussion by ScottChapman** [loki4j/loki-logback-appender#227] [Q&A]
This would be really useful for development/testing. You can see the tls_config for the agent [here](https://grafana.com/docs/agent/latest/flow/reference/components/loki.write/#tls_config-block)

**Comment by nehaev** [comment]
Hi @ScottChapman, yeah this makes sense. I'm open for a PR.

**Reply by ScottChapman** [reply]
It probably better to provide a general SSL configuration mechanism to support whatever folks might need to do in that space. Is that not possible with the current implementation? I think it is generally something that can be configured in the java http client

**Reply by nehaev** [reply]
I guess it should be possible. Just keep in mind that any new feature must be thoroughly tested to make sure it works as expected.

**Comment by ScottChapman** [comment]
Similarly is it possible to leverage our own CA if we are running Loki on a corporate network (using our own certs)?
